### PR TITLE
Update railties requirement in roadie-rails

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
 Group: Development/Languages
 License: MIT
 URL: https://github.com/Mange/roadie-rails
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(railties) >= 5.1
-Requires: %{?scl_prefix}rubygem(railties) < 5.3
+Requires: %{?scl_prefix}rubygem(railties) >= 6.0
+Requires: %{?scl_prefix}rubygem(railties) < 7.0
```
Comment (tbrisker) [inline packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:17]: This doesn't seem to match the gemspec - https://github.com/Mange/roadie-rails/blob/master/roadie-rails.gemspec#L25

File: packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
 Requires: %{?scl_prefix}rubygem(roadie) >= 3.1
 Requires: %{?scl_prefix}rubygem(roadie) < 4.0
```
Comment (tbrisker) [inline packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:19]: this too, should be <5.0

File: packages/foreman/rubygem-actionpack/rubygem-actionpack.spec [hunk 3]
```diff
@@ -18,9 +18,7 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.5.0
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(activesupport) = 6.0.2.2
-Requires: %{?scl_prefix_ruby}rubygem(rack) >= 2.0
-Requires: %{?scl_prefix_ruby}rubygem(rack) < 3
-Requires: %{?scl_prefix_ruby}rubygem(rack) >= 2.0.8
+Requires: %{?scl_prefix}rubygem(rack) >= 2.0.8
 Requires: %{?scl_prefix}rubygem(rack-test) >= 0.6.3
```
Comment (tbrisker) [inline packages/foreman/rubygem-actionpack/rubygem-actionpack.spec:22]: missing :
```
Requires: %{?scl_prefix}rubygem(rack) < 3

**Comment by ekohl** [comment]
For what it's worth, repoclosure failures look unrelated.

File: packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec [hunk 4]
```diff
@@ -14,9 +14,9 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: %{?scl_prefix_ruby}ruby(release)
```
Comment (ekohl) [inline packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:14]: Perhaps the spec file should be updated to allow for automatic updating?

Comment (zjhuntin) [reply packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:14]: sorry, not sure I follow, do you mean the template spec file?

Comment (ekohl) [reply packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:14]: These 2 lines allow it:

https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L32
https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L62

It also needs to know which template to use to generate the content:
https://github.com/theforeman/foreman-packaging/blob/7cdee5d764539d5342560c3b4631dd216d6aa31f/gem2rpm/scl.spec.erb#L1

Then in the future if you use `bump_rpm.sh` it'll automatically bump the dependencies. I started the conversion of a lot of the spec files, but didn't complete it.

Comment (zjhuntin) [reply packages/foreman/rubygem-roadie-rails/rubygem-roadie-rails.spec:14]: Ah okay, I'll go ahead and update it.

**Pull request merged**